labelbox/schema/dataset.py

```diff
@@ -58,12 +60,75 @@ class Dataset(DbObject, Updateable, Deletable):
         deprecation_warning=
         "This method does not return any data for batch-based projects and it will be deprecated on or around November 1, 2023."
     )
-    data_rows = Relationship.ToMany("DataRow", False)
     created_by = Relationship.ToOne("User", False, "created_by")
     organization = Relationship.ToOne("Organization", False)
     iam_integration = Relationship.ToOne("IAMIntegration", False,
                                          "iam_integration", "signer")
 
+    def data_rows(
+        self,
+        from_cursor: str = None,
+        where: Comparison = None,
+    ) -> PaginatedCollection:
+        """ 
+        Custom method to paginate data_rows via cursor.
+
+        Params:
+            after (str): Cursor (data row id) to start from, if none, will start from the beginning
+            where (dict(str,str)): Filter to apply to data rows. Where value is a data row column name and key is the value to filter on.    
+                example: {'external_id': 'my_external_id'} to get a data row with external_id = 'my_external_id'
+
+
+        NOTE: 
+            Order of retrieval is newest data row first.
+            Deleted data rows are not retrieved.
+            Failed data rows are not retrieved.
+            Data rows in progress *maybe* retrieved.
+        """
+
+        page_size = 500  # hardcode to avoid overloading the server
+        datarow_selections = query.results_query_part(Entity.DataRow)
+
+        empty_string = ''
+
+        where_clause = empty_string
+        where_vars = empty_string
+        where_param = None
+        if where is not None:
+            where_clause = ', $where: DatasetDataRowWhereInput'
+            where_vars = ', where: $where'
+            where_param = query.where_as_dict(Entity.DataRow, where)
+
+        template = Template(
+            """query DatasetDataRowsPyApi($$id: ID!, $$from: ID, $$first: Int$where_clause)  {
+                        datasetDataRows(id: $$id, from: $$from, first: $$first$where_vars)
+                            {
+                                nodes { $datarow_selections }
+                                pageInfo { hasNextPage startCursor }
+                            }
+                        }
+                    """)
```

Comment:
wouldn't this code read better if you would define all the arguments of `datasetDataRows` and pass `null` to those that are not specified? Instead of concatenating strings based on conditions.